src/tasks/components/TaskHeaderCard.tsx

```diff
@@ -0,0 +1,162 @@
+> {
+  public render() {
+    const {task} = this.props
+
+    if (!task) {
+      return null
+    }
+
+    return (
+      <ResourceCard
+        testID="task-card"
+        disabled={!this.isTaskActive}
+        alignItems={AlignItems.Center}
+        margin={ComponentSize.Large}
+        direction={FlexDirection.Row}
+        justifyContent={JustifyContent.SpaceBetween}
+      >
+        <FlexBox
+          alignItems={AlignItems.FlexStart}
+          direction={FlexDirection.Column}
+          margin={ComponentSize.Large}
+        >
+          <ResourceCard.Name name={task.name} testID="task-card--name" />
+          <ResourceCard.Meta>
+            {this.activeToggle}
+            <>Created at: {task.createdAt}</>
+            <>Created by: {task.name}</>
+            <>Last Used: {moment(task.latestCompleted).fromNow()}</>
+            <>{task.org}</>
+          </ResourceCard.Meta>
+        </FlexBox>
+
+        <FlexBox margin={ComponentSize.Medium} direction={FlexDirection.Row}>
+          <Button onClick={this.handleRunTask} text="Run Task" />
+          <Button
+            onClick={this.handleEditTask}
+            text="Edit Task"
+            color={ComponentColor.Primary}
+          />
+        </FlexBox>
+      </ResourceCard>
+    )
+  }
+
+  private get activeToggle(): JSX.Element {
+    const labelText = this.isTaskActive ? 'Active' : 'Inactive'
+    return (
+      <FlexBox margin={ComponentSize.Small}>
+        <SlideToggle
+          active={this.isTaskActive}
+          size={ComponentSize.ExtraSmall}
+          onChange={this.changeToggle}
+          testID="task-card--slide-toggle"
+        />
+        <InputLabel active={this.isTaskActive}>{labelText}</InputLabel>
+      </FlexBox>
+    )
+  }
+
+  private handleRunTask = () => {
+    const {onRunTask, match, getRuns} = this.props
+    try {
+      onRunTask(match.params.id)
+      getRuns(match.params.id)
+    } catch (error) {
+      console.error(error)
+    }
+  }
+
+  private handleEditTask = () => {
+    const {
+      history,
+      currentTask,
+      match: {
+        params: {orgID},
+      },
+    } = this.props
+
+    history.push(`/orgs/${orgID}/tasks/${currentTask.id}/edit`)
+  }
+
+  private get isTaskActive(): boolean {
+    const {task} = this.props
+    if (task.status === 'active') {
+      return true
+    }
+    return false
+  }
+
+  private changeToggle = () => {
+    const {task, onActivate} = this.props
+    if (task.status === 'active') {
+      task.status = 'inactive'
+    } else {
+      task.status = 'active'
+    }
+    onActivate(task)
+  }
+}
+
+const mstp = (state: AppState) => {
+  const {runs, runStatus, currentTask} = state.resources.tasks
+
+  return {
+    runs,
+    runStatus,
+    currentTask,
+  }
+}
+
+const mdtp = {
+  onAddTaskLabel: addTaskLabel,
+  onDeleteTaskLabel: deleteTaskLabel,
+  onRunTask: runTask,
+  getRuns,
+}
+
+const connector = connect(mstp, mdtp)
+
+export default connector(withRouter(TaskHeaderCard))
```

Comment:
like we mentioned in Zoom, change to named export. [Example here](https://github.com/influxdata/ui/blob/master/src/templates/containers/CommunityTemplatesIndex.tsx#L300)